src/adapter.js

```diff
@@ -350,7 +350,7 @@ function getId (s) {
 
 function getSpecsByName (specs, name) {
   specs = specs.filter(function (s) {
-    return s.name.indexOf(name) !== -1
+    return s.name.startsWith(name)
```

Comment:
@johnjbarton not sure what the supported browsers are but https://caniuse.com/#search=startswith

Keep that in mind in case this breaks some stuff since I notice old syntax used in the project :)

Reply:
yes this file is deliberately old school for maximum breadth of use.

Reply:
I think this PR could be adjusted to deal with the older API. 

But it also seem to me that the existing solution of matching anywhere has advantages.
A case like
```
test1
  subTest
test2
  subTest
```
can use 'test2 subTest' and succeed, while also using `subTest` to run both.

Reply:
Why not use a regex test which would work everywhere?

Reply:
> can use 'test2 subTest' and succeed, while also using subTest to run both.

You are right @johnjbarton, I was changing the suite directly in the url and did not notice that the link redirected to the full name of the suite.